open_learning_ai_tutor/Intermediary.py

```diff
@@ -1,6 +1,7 @@
-import open_learning_ai_tutor.PromptGenerator as PromptGenerator
+from open_learning_ai_tutor.prompts import get_problem_prompt, get_intent_prompt
```

Comment:
The plan is to remove Intermediary.py but leaving it for now until i refactor Tutor.py so i don't have to change Tutor.py  in this pr